src/robot_translation_config.example.json

```diff
@@ -1,3 +0,0 @@
-{
```

Comment:
Confused as to why the example config file needs to change for integration? Anyone using the robot_translation node will create their own config file and enter appropriate values in it -- this is just the example listing what values should be present.